tests/regression_tests/external_moab/test.py

```diff
@@ -0,0 +1,281 @@
+from pathlib import Path
+import os
+import shutil
+import subprocess
+import textwrap
+import glob
+from itertools import product
+import openmc
+import openmc.lib
+import numpy as np
+
+import pytest
+
+from tests.regression_tests import config
+from tests.testing_harness import PyAPITestHarness
+
+from subprocess import CalledProcessError
```

Comment:
Preferred import groupings from PEP8:
```suggestion
from pathlib import Path
import os
import shutil
import subprocess
from subprocess import CalledProcessError
import textwrap
import glob
from itertools import product

import openmc
import openmc.lib
import numpy as np
import pytest

from tests.regression_tests import config
from tests.testing_harness import PyAPITestHarness
```